src/cnv/ugbio_cnv/cnv_consts.py

```diff
@@ -0,0 +1,100 @@
+"""Constants for CNV module including INFO and FILTER tag registries."""
+
+# NOTE: both the id and the column name should appear in the registry
+# (after converting table to BED the names are as in the VCF)
+INFO_TAG_REGISTRY: dict[str, tuple[str, int | str, str, str, str]] = {
+    "CNV_calls_source": (
+        "CNV_SOURCE",
+        ".",
+        "String",
+        "the tool that called this CNV. can be combination of: cn.mops, cnvpytor, gridss",
+        "INFO",
+    ),
+    "CNV_SOURCE": (
```

Comment:
isn't this duplication for CNV_calls_source ?

Reply:
no, in cn.mops it is called CNV_calls_source, just trying not to change too much